openfisca_us/tests/policy/baseline/hhs/tanf/integration.yaml

```diff
@@ -0,0 +1,54 @@
+- name: "Example 1 from IL TANF website: Two-parent one-child family with $1,000 monthly income plus SSI newly applying."
```

Comment:
These cases from the website examples are great tests!

Reply:
Thanks! I'll try to create some other integration tests to test people that have a continuing application, so that I can test continuous TANF eligibility.